changelogs/fragments/996-postgresql_privs_fix_function_handling.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- postgresql_privs - fix the module mistakes a procedure for a function (https://github.com/ansible-collections/community.general/issues/994).
```

Comment:
Should this point to the current PR or the original one?

Reply:
The general rule is to put an issue link there. When an issue doesn't exist, we should put a link to a PR.

Reply:
I meant rather the ID (994 vs 996)

Reply:
yes, i understand, this is correct because we have the issue in this case

Reply:
In any case, it should be the exact same text as in the original PR. Backports shouldn't modify anything that's not absolutely needed (to resolve conflicts, or adjust to differences between the verison of the code in `stable-x` vs. `main`).